### DISCUSSION TITLE
Detecting field values without quoting

### DISCUSSION THREAD

**Discussion by IchigoKurasaki** [JoshClose/CsvHelper#1510] [Q&A]
I have csv file, where every field value in quotes and I need to detect the values of the fields without quotes to throw exception if this is found.
How can I detect these fields?
Thank you

**Comment by AltruCoder** [comment]
The answer here might give you some ideas on how you could detect fields that don't have quotes. https://stackoverflow.com/questions/61203650/how-to-pass-quotes-from-csv-to-writed-file-by-csvhelper
